src/main/java/tds/testpackageconverter/converter/mappers/CsidEnhancer.java

```diff
@@ -0,0 +1,60 @@
+package tds.testpackageconverter.converter.mappers;
+
+import org.opentestsystem.shared.contentspecid.ContentSpecId;
+import org.opentestsystem.shared.contentspecid.ContentSpecIdConverter;
+import org.opentestsystem.shared.contentspecid.enums.ContentSpecFormat;
+import org.opentestsystem.shared.contentspecid.enums.ContentSpecGrade;
+import org.opentestsystem.shared.contentspecid.enums.ContentSpecSubject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Optional;
+
+/**
+ * Enhance legacy format IDs to enhanced format content spec IDs. Since the legacy IDs in the test
+ * packages don't contain specification version information, it is assumed that when the subject is Math,
+ * the IDs are in MATH, v6 format, and if ELA, then the IDs are in ELA, v1. (For ELA, v1 is the only current
+ * legacy format.)
+ *
+ * The grade level also must be provided because some legacy IDs omit this information.
+ */
+public class CsidEnhancer {
```

Comment:
I think this should be renamed to ContentSpecIdConverter.  `Csid` isn't a common acronym used to refer to this id and spelling it out doesn't seem to wordy to me.